api/URLSearchParams.json

```diff
@@ -852,9 +905,15 @@
             "ie": {
               "version_added": false
             },
-            "nodejs": {
-              "version_added": "7.5.0"
-            },
+            "nodejs": [
+              {
+                "version_added": "7.3.0"
+              },
+              {
+                "version_added": "6.13.0",
+                "version_removed": "7.0.0"
+              }
+            ],
```

Comment:
Same as `entries()`, `forEach()`, and `keys()`, `values()` was only added in 7.3.0.

Reply:
see https://github.com/mdn/browser-compat-data/pull/24998#discussion_r1839777962